package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "start": "nodemon src/server.js",
     "electron": "electron .",
-    "build": "yarn && npm run check-cert && rimraf build && mkdir -p build && node bin/build.js && node bin/package.js",
+    "build": "yarn && npm run check-cert && rimraf build && mkdirp build && node bin/build.js && node bin/package.js",
```

Comment:
Is `mkdirp` provided by node or is installed as a package? (I don't have mkdirp on my system)

Reply:
Nevermind, it seems to work. But don't know where mkdirp comes from...

Reply:
Yup, `node_modules`.